package/harvester-os/files/usr/sbin/harv-install

```diff
@@ -5,6 +5,13 @@ ISOMNT=/run/initramfs/live
 TARGET=/run/cos/target
 DATA_DISK_FSLABEL="HARV_LH_DEFAULT"
 
+# Update environment variables
```

Comment:
`elemental` uses different environment variables compare to `HarvesterConfig.Install`, so we need to make some adaptations here. In the future, we would probably move these into `config.go`.